ui/src/components/ConnectionDataTable/index.jsx

```diff
@@ -209,6 +215,9 @@ export default class ConnectionDataTable extends Component {
   };
 
   handleFilterValueChange = e => {
+    const { history } = this.props;
```

Comment:
I think we can add `history` to the `propTypes`, as we are using it here. Not sure if we have linting rules covering this.

Reply:
Okay